packages/widget-chat/src/modules/redux-spark/spark.test.js

```diff
@@ -1,8 +0,0 @@
-import {createSpark} from './spark';
```

Comment:
Importing the actual spark added ~30 seconds to the test framework. We could probably mock out all of the plugins and test if we want to get full coverage. For now, I've removed the basic test.